ejb-multi-server/app-main/ejb/src/main/java/org/jboss/as/quickstarts/ejb/multi/server/app/MainAppBean.java

```diff
@@ -38,6 +40,7 @@
  * @author <a href="mailto:wfink@redhat.com">Wolf-Dieter Fink</a>
  */
 @Stateless
+@SecurityDomain("other")
```

Comment:
This change obviously works and was present on the beans in the app-twoA/app-twoB servers. What is puzzling is that the ejb3 subsystem has an attribute called "default-security-domain" with value "other" and according to the definition of the attribute, it should set the security domain of beans without a security domain to "other" - which is effectively what this annotation is doing. I suspect that there is something fishy with the operation of this attribute.

Reply:
Ok, so I'll keep that introduced definition